package WebGame.Server.Stock;
message InnerMessage {
  enum traits { value = 2 ;}
  required bytes information = 1;
}
